spec/api-spec.json

```diff
@@ -0,0 +1,111 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Sic-Code-API",
```

Comment:
If this is to follow the naming of the service example, perhaps:
sic-code-api
(the service names here always seem to be lowercase).

Reply:
updated to lowercase